README.md

```diff
@@ -42,14 +42,14 @@ To run this app, you require the following:
 
 - Kubernetes 1.11+
 - CoreDNS installed as [Cluster DNS Provider](https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/) (versions 1.3+ supported)
-- Helm v2 with a [working tiller service account](https://helm.sh/docs/using_helm/#installing-tiller)
+- Helm v3
```

Comment:
So we won't be compatible with `v2` at all anymore?

Reply:
No, the helm v3 chart changes are not backwards compatible AFAIK